<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomeDialog</class>
 <widget class="QDialog" name="WelcomeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>197</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Welcome to OpenMeta</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../OpenMeta/OpenMeta-analyst-/images/icons.qrc">
    <normaloff>:/images/meta.png</normaloff>:/images/meta.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>new dataset</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="dataset_name_le"/>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>what sort of data will you be working with?</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="bin_radio">
          <property name="text">
           <string>continuous and/or binary</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string>buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QRadioButton" name="diag_radio">
          <property name="text">
           <string>diagnostic</string>
          </property>
          <attribute name="buttonGroup">
           <string>buttonGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="chk_show">
       <property name="font">
        <font>
         <family>Verdana</family>
        </font>
       </property>
       <property name="text">
        <string>show this window on start-up</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="open_btn">
         <property name="text">
          <string>open ...</string>
         </property>
         <property name="icon">
          <iconset resource="../OpenMeta/OpenMeta-analyst-/images/icons.qrc">
           <normaloff>:/images/function_icon_set/folder_48.png</normaloff>:/images/function_icon_set/folder_48.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="open_recent_btn">
         <property name="font">
          <font>
           <family>Verdana</family>
          </font>
         </property>
         <property name="text">
          <string>open recent</string>
         </property>
         <property name="icon">
          <iconset resource="../OpenMeta/OpenMeta-analyst-/images/icons.qrc">
           <normaloff>:/images/function_icon_set/folder_48.png</normaloff>:/images/function_icon_set/folder_48.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="create_new_btn">
         <property name="text">
          <string>create new</string>
         </property>
         <property name="icon">
          <iconset resource="../OpenMeta/OpenMeta-analyst-/images/icons.qrc">
           <normaloff>:/images/function_icon_set/add_48.png</normaloff>:/images/function_icon_set/add_48.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images/icons.qrc"/>
  <include location="../OpenMeta/OpenMeta-analyst-/images/icons.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
